src/System.Management.Automation/engine/NativeCommandParameterBinder.cs

```diff
@@ -188,62 +188,83 @@ private void AppendOneNativeArgument(ExecutionContext context, object obj, Array
                     }
                 }
 
-                if (!string.IsNullOrEmpty(arg))
+                if (!ExperimentalFeature.IsEnabled("PSNativePSPathResolution") &&
```

Comment:
Is this the right experimental feature to check?  If so, can you explain the intent of this?

Reply:
Oops, no!